seismicpro/src/refractor_velocity/refractor_velocity_field.py

```diff
@@ -145,6 +150,79 @@ def __str__(self):
 
         return msg
 
+    @classmethod  # pylint: disable-next=too-many-arguments
+    def from_survey(cls, survey, is_geographic=None, auto_create_interpolator=True, init=None, bounds=None,
+                    n_refractors=None, min_velocity_step=1, min_refractor_size=1, loss='L1', huber_coef=20, tol=1e-5,
+                    first_breaks_col=HDR_FIRST_BREAK, bar=True, **kwargs):
+        """Create a field by fitting a near-surface velocity model for each gather in the survey.
```

Comment:
You use fit here and estimate in the bar desc, please unify.
```suggestion
        """Create a field by estimating a near-surface velocity model for each gather in the survey.
```